frontend/src/v5/ui/routes/routes.constants.ts

```diff
@@ -14,9 +14,19 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import { matchPath } from 'react-router';
 
 export const NOT_FOUND_ROUTE_PATH = '/v5/404';
-export const VIEWER_ROUTE = '/v5/viewer/:teamspace/:containerOrFederation/:revision?';
+
+export const VIEWER_ROUTE = '/v5/viewer/:teamspace/:project/:containerOrFederation/:revision?';
+export const DASHBOARD_ROUTE = '/v5/dashboard';
+export const PROJECTS_LIST_ROUTE = `${DASHBOARD_ROUTE}/:teamspace`;
+export const PROJECT_ROUTE = `${PROJECTS_LIST_ROUTE}/:project/t/:tab`;
+export const CONTAINERS_ROUTE = `${PROJECTS_LIST_ROUTE}/:project/t/containers`;
+export const FEDERATIONS_ROUTE = `${PROJECTS_LIST_ROUTE}/:project/t/federations`;
```

Comment:
PROJECT_ROUTE, CONTAINERS_ROUTE, and FEDERATIONS_ROUTE all share the same root, i.e `${PROJECTS_LIST_ROUTE}/:project/t`. Would it be worth it to create another constant for it?